docs/core_systems/service_registry.md

```diff
@@ -176,7 +176,6 @@ Auto initialization of database tables. Value must be always 'none'.
 :fontawesome-solid-wrench: **authentication.policy**
 
 Way of authenticating the service requester systems. Can be `declared`, `certificate` or `outsourced`.
```

Comment:
You already put this into the general config, You should delete from here as well.